tests/e2e/scenarios/test_reborn_webui_v2_smoke.py

```
@@ -353,6 +353,175 @@ async def test_reborn_v2_ui_send_renders_reply(reborn_v2_page, reborn_v2_server)
+            if (!stream) throw new Error("no EventSource stream is open");
+            const event = new MessageEvent(type, {
+              data: JSON.stringify({ type, ...frame }),
+              lastEventId: id,
+            });
+            stream.dispatchEvent(event);
+          };
+        })();
+        """
+    )
+
+    async def fulfill_json(route, body, status=200):
```

Ruff (0.15.18)

[warning] Missing return type annotation for private function `fulfill_json`

Add return type annotation: `None`

(ANN202)